### PR TITLE
fix margin #22

### PR DESCRIPTION
**Pull request by gosoccerboy5**
#22
before:

after:
Screen Shot 2021-05-09 at 4 25 15 PM

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -58,3 +58,6 @@ button {
 	background-color: transparent;
 	text-decoration: underline;
 	}
+.TOC h3 {
+	line-height: 20px;
```
Comment (leahcimto) [inline stylesheet.css]: Could you add this to .TOC since having a new section is unnecessary (I tested this)

**Review by leahcimto — CHANGES_REQUESTED** [review]

File: stylesheet.css (hunk 1 above)
Comment (gosoccerboy5) [reply stylesheet.css]: working on it

Comment (gosoccerboy5) [reply stylesheet.css]: finished

**Review by Accio1 — APPROVED** [review]
Looks good!

**Pull request merged**